modAionImpl/src/org/aion/zero/impl/pendingState/AionPendingStateImpl.java

```diff
@@ -280,140 +287,102 @@ private void addTransactionsFromCacheOrBackup(List<AionTransaction> transactions
         List<TxResponse> txResponses = new ArrayList<>();
```

Comment:
This method seems to be used both when the return type is necessary and when it's not. Could we have separate implementations for the two cases?